pkg/filters/args.go

```diff
@@ -42,14 +42,30 @@ func (filter *ArgFilter) Filter(eventID events.ID, args []trace.Argument) bool {
 			}
 		}
 		if !found {
-			return true
+			return false // always filter if argument does not exist
```

Comment:
Can we test only witrh this change and see that nothing breaks?

Reply:
Sure, let me check!

Reply:
Well only this change still brakes e2e because of symbols loaded event. Adding a mitigation for that, plus this change, is the way to go for this release (IMO).